src/main/java/io/vertx/ext/mongo/AggregateOptions.java

```diff
@@ -158,19 +158,25 @@ public AggregateOptions setAllowDiskUse(final Boolean allowDiskUse) {
   }
 
   @Override
-  public boolean equals(final Object o) {
-    if (this == o) {
-      return true;
-    }
-    if (o == null || getClass() != o.getClass()) {
-      return false;
-    }
-    final AggregateOptions that = (AggregateOptions) o;
-    return batchSize == that.batchSize && maxTime == that.maxTime && maxAwaitTime == that.maxAwaitTime && allowDiskUse == that.allowDiskUse;
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+    AggregateOptions that = (AggregateOptions) o;
+    return getBatchSize() == that.getBatchSize() && getMaxTime() == that.getMaxTime() && Objects.equals(getAllowDiskUse(), that.getAllowDiskUse()) && Objects.equals(getCollation(), that.getCollation());
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(batchSize, maxTime, maxAwaitTime, allowDiskUse);
+    return Objects.hash(getBatchSize(), getMaxTime(), getAllowDiskUse(), getCollation());
```

Comment:
There's no benefit in calling the getters.

Reply:
This is auto generated code (I will change it :disappointed: )